### DISCUSSION TITLE
XPlane has stopped entering VR

### DISCUSSION THREAD

**Discussion by wakevortex** [retroluxfilm/reshade-vrtoolkit#16] [Q&A]
The mod was working really well, but something must have changed in SteamVR or WMR..my reverb G2 will now NOT enter VR when the Vrtoolkit is in the xplane directory.. 
I have tried also on a clean X plane install ..same problem..it APPEARS to enter vr , but nothing now appears  in the goggles..(no xplane picture)  though it IS on the screen 
Remove the mod...back to normal...this is repeatable..
any ideas?  as I mentioned it WAS working well ....what a shame 

Bill P

**Comment by retroluxfilm** [comment]
Hmm not sure what is causing this. Did you update the vr toolkit version inbetween or did you stayed on the same version when it occured?

If you are in SteamVR beta you can try to get out of beta and recheck if it works then. Might be caused by an recent update of the SteamVR beta.

**Comment by wakevortex** [comment]
Thanks for fast reply... VRtoolkit version hasn't changed..I've just checked SteamVR and I'malready  opted out of the beta.
I WAS in beta for windows mixed reality  for steam VR ..just .opted out of that and I have a  new error now ! (when entering VR) 

 Render Setup Failed: Shared IPC Connect Failed After Multiple Attempts (308
 
again if I remove your mod it goes in perfectly to VR...all VERY odd , and this is on a fresh X plane install for testing to eliminate plugins etc

**Comment by retroluxfilm** [comment]
You could send me the openg32.log created where the reshade dll is located. There might be some clue there.

**Reply by wakevortex** [reply]
Ive attached that file..lets hope it throws some light on this strange issue ! ...appreciate it 
Bill P
[OPENGL32.log](https://github.com/retroluxfilm/reshade-vrtoolkit/files/7874708/OPENGL32.log)

**Comment by retroluxfilm** [comment]
I tested it with the Nvidia drivers I had installed and with the latest drivers you also used and it works in both cases for me.
I am on SteamVR beta using the index so no WMR is involved and used the latest Xplane 11 demo version.

So it might be connected to WMR that is causing your issue.

**Reply by wakevortex** [reply]
Hmm..that certainly narrows it down I guess. Not sure what I can do my end as of course I cant avoid using WMR ...I do appreciate you having a look anyway ...shame -  as I loved what your plugin did ! 
Bill P
